shared/modloader.h

```diff
@@ -60,6 +74,14 @@ MODLOADER_EXPORT extern JavaVM* modloader_jvm;
 MODLOADER_EXPORT extern void* modloader_libil2cpp_handle;
 /// @brief The captured dlopen-d libunity.so handle
 MODLOADER_EXPORT extern void* modloader_unity_handle;
+/// @brief True if libs have been dlopened
+MODLOADER_EXPORT extern bool libs_loaded;
```

Comment:
I think these should be renamed to `thing_opened` to be more clear about what they stand for, also the early_mods_loaded and late_mods_loaded are not actually what their comments describe them as, they are similiar to libs, set to true when opened, not when load was called for all of them